templates/search_page.html

```diff
@@ -182,6 +194,16 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
     <script type="module" src="assets/js/search.js"></script>
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+    <script>
+        $(document).ready(
```

Comment:
can we also move this function to `search.js`

Reply:
tried to move this out , but the function does not seem to be recognized after being moved @rwangsc18 , so decided to keep it here itself